components/encoder/fake/encoder.go

```diff
@@ -69,7 +71,8 @@ func (e *fakeEncoder) Reconfigure(
 
 // Config describes the configuration of a fake encoder.
 type Config struct {
-	UpdateRate int64 `json:"update_rate_msec,omitempty"`
+	UpdateRate  int64   `json:"update_rate_msec,omitempty"`
+	TicksPerSec float64 `json:"ticks_per_sec,omitempty"`
```

Comment:
Instead of this - and dealing with the zero default value that will have the driver stall, let's do a non-user facing default speed value at the top of this file.

You can also use the update rate multiplied by a default constant if you prefer.

Reply:
I don't think this pr should change the default behavior of fake encoders, that would require changing more of the tests.

Reply:
I don't think the tests are testing with any encoder functionality from a cursory look, so it can just be changed in the fake motor and gpio motor tests to injects.